Cargo.toml

```diff
@@ -12,10 +12,10 @@ license = "MIT"
 [dependencies]
 anyhow = "1"
 chrono = { version = "0.4", features = ["serde"] }
-mlmd = "0.2"
+mlmd = { version = "0.2", default-features = false, features = ["runtime-tokio-rustls"] }
 palette = "0.6"
 serde = {version = "1", features = ["derive"]}
 serde_json = "1"
 structopt = "0.3"
 tinytemplate = "1"
-tokio = {version = "1", features = ["macros"]}
\ No newline at end of file
+tokio = {version = "1", features = ["macros"]}
```

Comment:
Sorry, an unrelated change introduced due to my local environment. I can revert it if requested.

Reply:
No problem :)